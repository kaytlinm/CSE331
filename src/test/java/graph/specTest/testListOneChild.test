#List One Child Test File
#Tests that a graph can be created, a node pair connected by one edge can print out the singular child.

CreateGraph graph1
AddNode graph1 a
AddNode graph1 b
AddEdge graph1 a b e1
ListChildren graph1 a