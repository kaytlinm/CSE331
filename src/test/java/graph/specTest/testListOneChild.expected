#List One Child Test File
#Tests that a graph can be created, a node pair connected by one edge can print out the singular child.

created graph graph1
added node a to graph1
added node b to graph1
added edge e1 from a to b in graph1
the children of a in graph1 are: b(e1)